utilities/tools/Set-ModuleReadMe.ps1

```diff
@@ -649,6 +649,11 @@ function ConvertTo-FormattedJSONParameterObject {
         [string] $BicepParamBlock
     )
 
+    if ([String]::IsNullOrEmpty($BicepParamBlock)) {
```

Comment:
Kindly remove changes not in the scope of this PR

Reply:
Actually it was. It is in the title and description 😉

Reply:
Will merge set change back from main once that PR is merged

Reply:
I will merge back from main once that PR ist merged

Reply:
I will merge back from main once that PR ist merged

Reply:
I wish there'd be a native feature to set one PR as a predecessor to another. As this one would be blocked without the fix that comes with it. Meh... Maybe one day

Reply:
done